### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by ecorman**

### CODE REVIEW

**Comment by jamorham** [comment]
I haven't yet been able to get a review of this PR by a native speaker.

**Comment by ecorman** [comment]
Well... I'm native speaker and current translation had a lot of issues. Why not to go on and improve it next time?

File: app/src/main/res/values-es/strings-es.xml [hunk 1]
```diff
@@ -145,22 +145,22 @@
...
-	<string name="send_crash_errors_to_developer">Enviar a errores de choque desarrollador</string>
+	<string name="update_channel">Canal de actualización</string>
+	<string name="send_crash_errors_to_developer">Enviar a errores desarrollador</string>
 	<string name="automatic_crash_reporting">Informes de error automáticos</string>
-	<string name="send_feedback_message">Enviar Mensaje Comentarios</string>
-	<string name="copying_settings">Copia de ajustes</string>
-	<string name="show_settings_qr_codes">Mostrar códigos QR Ajustes</string>
-	<string name="load_save_settings_to_sdcard">Cargar / Guardar configuración para tarjeta SD</string>
+	<string name="send_feedback_message">Enviar Mensaje de feedback</string>
+	<string name="copying_settings">Ajustes de copia</string>
```
[the remark this answers is not in the document]
Comment (ecorman) [reply app/src/main/res/values-es/strings-es.xml]: You're right, I did a mistake here, then translation should be "Transferir Ajustes" or "Copiar Ajustes"? Previous translation was also good "Copia de Ajustes" I don't know why I changed it!

File: app/src/main/res/values-es/strings-es.xml [hunk 2]
```diff
@@ -250,69 +250,69 @@
...
+	<string name="xdrip_plus_security_key_settings_only">Sólo ajustes de clave de seguridad xDrip+</string>
+	<string name="show_general_and_collection_settings">Mostrar ajustes generales y de la colección</string>
 	<string name="copy_all_settings">Copiar todos los ajustes</string>
-	<string name="sensor_is_ready_please_enter_double_calibration">El sensor está listo, por favor, introduzca una doble calibración</string>
-	<string name="in_order_to_get_started_please_perform_two_finger">Para empezar, por favor realizar dos pruebas de pinchazo en el dedo e introduzca los valores aquí!</string>
-	<string name="enter_first_bg_value">Primero Introduzca el valor de glucosa en sangre</string>
-	<string name="enter_second_bg_value">Introduce segundo valor de glucosa en sangre</string>
+	<string name="sensor_is_ready_please_enter_double_calibration">El sensor está listo, por favor, introduzca calibración inicial</string>
+	<string name="in_order_to_get_started_please_perform_two_finger">Para empezar, por favor realice dos test de pinchazo en el dedo e introduzca los valores aquí!</string>
+	<string name="enter_first_bg_value">Entre Primer Valor BG</string>
```
Comment (pedroll) [inline app/src/main/res/values-es/strings-es.xml]: Introduzca Primer valor de BG

File: app/src/main/res/values-es/strings-es.xml [hunk 3]
```diff
@@ -250,69 +250,69 @@
...
+	<string name="enter_second_bg_value">Entre Segundo Valor BG</string>
```
Comment (pedroll) [inline app/src/main/res/values-es/strings-es.xml]: Introduzca Segundo valor de BG

File: app/src/main/res/values-es/strings-es.xml [hunk 4]
```diff
@@ -359,124 +358,317 @@
...
 	<string name="snooze_alert_before_it_fires">Silenciar alerta antes del disparo</string>
 	<string name="create_low_alert">Crear alerta de baja</string>
 	<string name="create_high_alert">Crear alerta de alta</string>
 	<string name="long_press_an_existing_alert_to_edit">(Pulsación larga para editar una alerta existente)</string>
 	<string name="adding">Añadiendo</string>
 	<string name="high">alto</string>
 	<string name="alert">alerta</string>
-	<string name="high_alert">Alerta alta</string>
-	<string name="low_alert">Alerta de baja</string>
+	<string name="high_alert">Alerta hiperglucemia</string>
```
Comment (pedroll) [inline app/src/main/res/values-es/strings-es.xml]: outside the alarm limits is not always Hyperglycemia / Hypoglycemia but High / low

**Comment by CaroGo** [comment]
Is there still any help needed with the translation? I am "half-native" ;-) Sometimes it is really difficult to find the appropiate expression and in a short way...even in german!

File: app/src/main/res/values-es/strings-es.xml (hunk 2 above)
Comment (ecorman) [reply app/src/main/res/values-es/strings-es.xml]: Creo que no cabe quizá podría ser "Primer Valor BG" sin más

**Comment by ecorman** [comment]
I don't know how to change the sentences according the comments and it seems some approval is required also to being able to merge the pull request.

**Comment by jamorham** [comment]
@ecorman you should push further commits updating the changes based on comments made by commentators if you agree with their suggestions or discuss those items further if not.

**Comment by ecorman** [comment]
Why this is not merged? Should I do something? Github is not easy for me. Can anybody help on this?

**Comment by IreneMagica777** [comment]
The approval requires one more positive review, I'll do It, but at least consider my suggestión...

**Comment by ecorman** [comment]
I deleted my local xdrip source code copy and now I don't know how to include your suggestion. But you can create later a pull request and then I'll approve it. I think I can do that in github website.

**Comment by jasom77** [comment]
As ecorman deleted his repository I am going to approve this translation and then i will make the previous suggestions about it.

**Pull request merged**